### ISSUE TITLE
Segmentation fault with gtk3 3.22.29+60+ge42d8598ca

### ISSUE THREAD

**Issue by falkse** [CrimsonAS/gtkplatform#36]
Distribution: Arch Linux x64
GTK Version: gtk3 3.22.29+60+ge42d8598ca-1
Qt Version: 5.10.1

Problem: With the latest GTK update the gtkplatform Qt module is broken. Applications crash with a segfault.

How to reproduce:
1. Launch Qt5 Application with environment QT_QPA_PLATFORM=gtk (e.g. keepassx, telegram-desktop)
2. See the error:
```
(keepassxc:5600): Gdk-CRITICAL **: 14:17:18.802: gdkdisplay-wayland.c:1317: Truncating shared memory file failed: Invalid argument
Segmentation fault
```

Going back to gtk3-3.22.29+4+gb485cf91b5 fixes the problem.

**Comment by falkse** [comment]
Issue resolved with latest update 3.22.29+86+gd92df4ff2c.
Can get closed.

**Comment by rburchell** [comment]
Thanks, good to know.
